### PR TITLE
backend: Travis CI Integrated

### PR DESCRIPTION
**Pull request by iliyanmotovski**
Automatic CI builds with travis integrated for the backend.
Had to change some tests to work with UTC time format,
due to travis errors, when working with EET.

### CODE REVIEW

**Comment by iliyanmotovski** [comment]
Ready for review.

File: .netrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+machine github.com
```
Comment (mgenov) [inline .netrc]: It's a public project so this information should not be required.

File: bank/persistence/authetications_test.go [hunk 2]
```diff
@@ -30,8 +30,9 @@ func TestItStartsUpdatesAndDeletesSession(t *testing.T) {
 
 	result, _ := sessionStore.FindSessionAvailableAt(userSession.SessionID, startInstant)
 
-	if *result != expected {
-		t.Errorf("expected session: %v got: %v", expected, result)
+	if result.UserID != expected.UserID || result.SessionID != expected.SessionID || result.Expires.UTC() != expected.Expires.UTC() {
```
Comment (mgenov) [inline bank/persistence/authetications_test.go]: You can use reflect.DeepEqual for comparison if you want to compare everything.

**Review by mgenov — CHANGES_REQUESTED** [review]

**Comment by iliyanmotovski** [comment]
Ready for review.

**Review by mgenov — APPROVED** [review]
LGTM !

**Pull request merged**